Fix spelling errors

chosungmann opened the pull request:
aligment -> alignment
constructable -> constructible
constuctible -> constructible
contructed -> constructed
destructable -> destructible
futher -> further
implcit -> implicit
implemenation -> implementation
intrinics -> intrinsics

mbxx commented on absl/time/internal/cctz/include/cctz/civil_time.h:
```diff
@@ -150,7 +150,7 @@ namespace cctz {
 // All civil-time types have accessors for all six of the civil-time fields:
 // year, month, day, hour, minute, and second. Recall that fields inferior to
-// the type's aligment will be set to their minimum valid value.
+// the type's alignment will be set to their minimum valid value.
```
This file is actually pulled from the CCTZ, which lives in another GitHub repo (https://github.com/google/cctz/blob/master/include/cctz/civil_time.h).  Would you mind reverting this file and making this change upstream?  Abseil will pick up the change when we update our copy of CCTZ.

mbxx reviewed (commented):
Thank you!  Looks great, but if you wouldn't mind, it would be better to address the misspelling in absl/time/internal/cctz/include/cctz/civil_time.h upstream (I left a comment pointing at https://github.com/google/cctz/blob/master/include/cctz/civil_time.h, which is the upstream source of the file)

chosungmann committed:
Fix spelling errors

aligment -> alignment
constructable -> constructible
constuctible -> constructible
contructed -> constructed
destructable -> destructible
futher -> further
implcit -> implicit
implemenation -> implementation
intrinics -> intrinsics

ghost commented:
@mbxx Thanks for the review! I opened a pull request to address your comment at https://github.com/google/cctz/pull/127 Please take a look.

CJ-Johnson commented:
Since https://github.com/google/cctz/pull/127 was merged, I'm closing this. Thank you for the contribution!

ghost commented:
@CJ-Johnson I’m just wondering why this PR has been closed without merging?

CJ-Johnson commented:
Thank you for the ping! I misread the comment stream and understood https://github.com/google/cctz/pull/127 to be the entire scope of this PR. That was incorrect. My apologies, for that.

I will work on merging this, now.
